res/sw.js

```diff
@@ -1 +0,0 @@
-self.addEventListener("fetch", () => {});
```

Comment:
this might be a stupid question, but why do we currently have a service worker that is essentially a no-op?